README.md

```diff
@@ -15,35 +21,5 @@
     npm run build
     npm run start:prod
 ```
-    
-## Docker
```

Comment:
you don't need to delete this section. It works fine, also for `start:prod`.

Just update documentation: 
set SMTP variables in `.env` after running `npm run start` 

```batch
SMTP_HOST=smtp.ethereal.email
SMTP_PORT=587
SMTP_USER=<autogenerated>
SMTP_PASS=<autogenerated>
```

```batch
docker compose build
docker compose up
```